src/components/map/controls/layer-toggle/component.jsx

```diff
@@ -1,26 +1,48 @@
 import React from 'react';
-import './styles.scss';
+import ReactTooltip from 'react-tooltip';
+import PropTypes from 'prop-types';
 
-function LayerToggle({layers, setLayers}) {
+import Icon from 'components/icon';
+import LayersToggleModal from './modal';
+
+import styles from './styles.module.scss';
+
+const LayerToggleComponent = ({ tooltipRef, toggleLayerActive, setToggleLayerActive }) => {
   return (
-    <div className="c-layer-toggle">
-      {layers && layers.map((l, i) =>
-        <p key={`layer-${i}`}>
-          Layer {i+1}
-          <input
-            type="checkbox"
-            checked={l.active}
-            onChange={() =>
-              setLayers(layers.map(layer =>
-                layer.id !== l.id ?
-                layer :
-                {...l, active: !l.active}
-              ))
-            }></input>
-        </p>
-      )}
+    <div className={styles.layerToggle}>
+      <button
+        data-for="layerTootlip"
+        className={styles.mapNavigationButton}
+        data-tip=""
+        style={{ filter: toggleLayerActive ? 'invert(1)' : 'unset' }}
+      >
+        <Icon name="icon-stack" />
+      </button>
+      <ReactTooltip
+        id="layerTootlip"
+        ref={tooltipRef}
+        clickable
+        className="layerTooltip"
+        effect="solid"
+        globalEventOff="click"
+        event="click"
+        place="right"
+        afterShow={() => setToggleLayerActive(true)}
+        afterHide={() => setToggleLayerActive(false)}
+        overridePosition={({ top, left }) => {
+          return { top: top + 98, left };
```

Comment:
could you depend that `98` value off of the parent so if we decide to move the toolbox it's gonna break the layout?